addons/killtracker/stringtable.xml

```diff
@@ -86,5 +90,13 @@
             <Japanese>同士討ち</Japanese>
             <Turkish>Dost Atışı</Turkish>
         </Key>
+        <Key ID="STR_ACE_KillTracker_TrackAI_DisplayName">
+            <English>Track AI units killed by player</English>
+            <Czech>Sledovat AI zabité hráči</Czech>
+        </Key>
+        <Key ID="STR_ACE_KillTracker_TrackAI_Description">
+            <English>Defines if killed AIs will be shown in the kill tracker during mission debriefing</English>
```

Comment:
```suggestion
            <English>Defines if killed AIs will be shown in the kill tracker during mission debriefing.</English>
```

Reply:
@ilbinek Do the same with the Czech translation, if it's also considered to be a sentence in Czech.

Reply:
I would call it a sentence, edited.